config.js

```diff
@@ -1163,6 +1165,16 @@ var config = {
     // will open an etherpad document.
     // etherpad_base: 'https://your-etherpad-installati.on/p/',
 
+
+    // Generic IFrame to be used for any application you like. You can name the menu entries in "/lang/main.json"
+    // The link supports template strings for:
+    //  - {room} - The room id
+    //  - {lang} - The ISO Language tag
+    //
+    // sharedIFrameTemplateUrl: 'https://example.com/link/to/your/iframe/app/{room}?lang={lang}',
+    // sharedIFrameName: 'Whiteboard',
```

Comment:
Can you use sharedIFrameName for the button name, so it shows 'Start Whiteboard'?

Reply:
Sure, i will modify the buttons.

Reply:
This is now configurable in the main language files, based on the key used for the iframe, see: 
https://github.com/jitsi/jitsi-meet/pull/11052/commits/0d41f018ec0182f621772ab2d72efc27c218db4f#diff-acc6f5186c27e0f1c972135d39d2567f7deea1a3993c13eb8df685a955caa869

Reply:
Do you see a problem with dynamically using the name coming form the config?
You cannot use the language files without rebuilding. And this is being configured from config.js ... If my service is PacMan in iframe, I cannot set a name without rebuilding the sources ...

Reply:
Ugh, I thought it would be at runtime. I did not deploy the update to our server yet.

Reply:
Nope :) the main.json is embed in the app.bundle for web. On mobile all of the language files are embedded in the app.

Reply:
Ahh, this clears up another problem I had. Now I know why it does not update the english version of my app. 
But why is it used as english then (instead of as a fallback, when no other string is supplied)? 
So it would work in every language except English...
Should I go back to the stitching approach of using the participant name? and a prefix/suffix button string combination. 
I thought it would make more sense to supply the user with the actual word in their language instead of with the same in every language.

Reply:
> I thought it would make more sense to supply the user with the actual word in their language instead of with the same in every language.

There is no mechanism to add this dynamically at the moment.



> Should I go back to the stitching approach of using the participant name? (a prefix/suffix button string combination).

If you mean to generate the button name from the config string - yes.



> But why is it used as english then (instead of as a fallback, when no other string is supplied)?

I didn't get that.

Reply:
> There is no mechanism to add this dynamically at the moment.

For me it is already working. I tried again on our environment and all updates to the json files get applied without recompiling, except the one from the main.json. 

> If you mean to generate the button name from the config string - yes.

I will update as you suggested. 

>  > But why is it used as english then (instead of as a fallback, when no other string is supplied)?
>
> I didn't get that.

The main-en.json does not seem to get used. It always uses the english words from main.json. 
I thought the main.json strings are used as fallback, if a language does not specify a translation string.